pkgs/development/libraries/libxc/default.nix

```diff
@@ -1,32 +1,39 @@
-{ lib, stdenv, fetchurl, gfortran, perl }:
+{ lib, stdenv, fetchFromGitLab, cmake, gfortran, perl }:
 
 let
-  version = "4.3.4";
+  version = "5.1.0";
 
 in stdenv.mkDerivation {
```

Comment:
Can you remove the let in?

Reply:
I actually find that a more clean solution than having a recursive set.